CHANGELOG.md

```diff
@@ -1090,6 +1092,8 @@ https://docs.voltocms.com/upgrade-guide/
 ### Feature
 
 - Allowing user to paste url in search box in objectBrowser @iFlameing
+- Log in and out through either Plone Classic or the Volto UI also does the same in the
```

Comment:
I don't see the changelog entry matching the actual changes.

Reply:
Right you are, [pushed](https://github.com/plone/volto/commit/8d887af0e9c18b03effb86a99ade53f96bdec710).  Thanks for the catch.